File: Practice Problems/bubbleSort.java
boolean swapped = false;
for(int j=arr.length-1; j>=0 && swapped; j--)
{
swapped = true;
for(int k=0; k < j; k++)
{
if(arr[k] > arr[k+1])
{
int temp = arr[k];
arr[k] = arr[k+1];
arr[k+1] = temp;
swapped = false;
}
}
}

File: Practice Problems/bubbleSort.js
function insertionSort(arr, array_size) {
    let i, j, value;
    for (i = 1; i < arr.length; i++) {
        value = arr[i];
        j = i;
        while ((j > 0) && (arr[j - 1] > value)) {
            arr[j] = arr[j - 1];
            j = j - 1;

        }
        arr[j] = value;
    }
    return value;
}
console.log(insertionSort([10, 9, 7, 6, 5, 4, 3, 8, 2, 1]));